src/core/executor.c

```diff
@@ -208,10 +208,10 @@ int main(int argc, char *argv[]) {
                 return r;
 
         /* Now try again if we were told it's fine to use a different target */
-        if (log_get_target() != LOG_TARGET_KMSG) {
+        if (log_get_target() != LOG_TARGET_KMSG)
                 log_set_prohibit_ipc(false);
```

Comment:
This conditionalization makes no sense to me. Can you explain @bluca? I think we should just drop the whole conditionalization

Reply:
if that is set to false and we are starting the journal, will things work? if so that's fine

Reply:
I can't parse this.

if what is false? and what's fine? /me is very confused by this